pkg/sqlmanager/reader.go

```diff
@@ -371,6 +387,23 @@ func getIntValue(value interface{}) (types.StringOrArray, error) {
 	}
 }
 
+func getBooleanValue(value interface{}) (types.StringOrArray, error) {
+	switch t := value.(type) {
+	case bool:
+		return types.NewStringValue(strconv.FormatBool(t)), nil
```

Comment:
Make sure that strconv.FormatBool does not return an uppercase "True" ;)

Reply:
I think the test checks this